tools/swift-format/Sources/Rules/UseEarlyExits.swift

```diff
@@ -15,5 +15,42 @@ import SwiftSyntax
 ///
 /// - SeeAlso: https://google.github.io/swift#guards-for-early-exits
 public final class UseEarlyExits: SyntaxFormatRule {
+  
+  public override func visit(_ node: CodeBlockSyntax) -> Syntax {
+    
+    var newItems: [CodeBlockItemSyntax] = []
+    for statement in node.statements {
+      guard let ifStmt = statement.item as? IfStmtSyntax else { return node }
+      guard let elseStmt = ifStmt.elseBody else { return node }
+      guard let elseBody = elseStmt as? CodeBlockSyntax else { return node }
 
+      if elseContainsControlStmt(elseStmt: elseStmt) {
+        guard let moveDeletedIfCode = visit(ifStmt.body.withLeftBrace(nil).withRightBrace(nil)) as? CodeBlockSyntax else { continue }
+        let formattedIfCode = reindentBlock(block: moveDeletedIfCode, adjustment: -2)
```

Comment:
You can avoid reindenting this block; the pretty-printer will take care of reindenting for you.